app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -65,7 +84,13 @@ const MakeDecision = ({ data }) => {
     });
   };
   const composeDataToSend = () => {
-    return [{ testItemId: itemData.id, issue: state.issue }];
+    return [
+      {
+        ...itemData,
```

Comment:
Are you sure we need all the item data for the `saveDefect` request?

Reply:
same logic used before